runtime/moonbase/src/lib.rs

```diff
@@ -241,7 +241,7 @@ where
 }
 
 parameter_types! {
-	pub const TransactionByteFee: Balance = 1;
+	pub const TransactionByteFee: Balance = currency::deposit(0, 1);
```

Comment:
What is this `deposit` function supposed to do? I see its definition but I don't understand what `items` and bytes` are. Maybe we could add some docs to it?

Reply:
Thanks for pointing that out. Tbh I used the function because it already calculates a 10 MILLICENT per byte fee. But now that I read your comment it's true that it's purpose is different: calculate the deposit for the Proxies.

Let's not mix things :) just rewrote that, defined a `BYTE_FEE` constant, and make both `TransactionByteFee` and `currency::deposit` use it.